XenAdminTests/CommandTests/MigrateVirtualDiskCommandTest.cs

```diff
@@ -50,7 +50,7 @@ public class MigrateVirtualDiskCommandTest : DatabaseTester_TestFixture
         [Test]
         public void CheckContextMenuText()
         {
-            MigrateVirtualDiskCommand cmd = new MigrateVirtualDiskCommand(mw, null as VDI);
+            var cmd = new MigrateVirtualDiskCommand(mw, null);

```

Comment:
Now that I'm looking at it again, this test is conceptually wrong because it's checking the text is correct on a command that shouldn't have been constructed at all due to null selection...